python/tchannel/tornado/message_factory.py

```diff
@@ -36,6 +41,156 @@ def __init__(self):
         # value: incomplete streaming messages
         self.message_buffer = {}
 
+    def build_raw_request_message(self, request, args, is_completed=False):
```

Comment:
What does the `raw` here imply? What's the relationship between `build_raw_request_message` and `build_request`?